src/barrier.rs

```diff
@@ -1,5 +1,4 @@
 use event_listener::{Event, EventListener};
-use futures_lite::ready;
```

Comment:
Here it was an unused import. We don't have a check for disallowing unused dep? :thinking:

Reply:
It was being used by the future implementation down lower; it's just been replaced with the `ready!` macro in the `lib.rs` so that the transition is seamless.